lt/client/client.go

```diff
@@ -291,17 +301,23 @@ func (u *User) transmitSpeech() {
 		log.Fatalf("%s: failed to create opus encoder: %s", u.cfg.Username, err.Error())
 	}
 
-	if err := u.callsClient.Unmute(track); err != nil {
-		log.Fatalf(err.Error())
-	}
-
 	for text := range u.speechTextCh {
 		func() {
 			defer func() {
+				time.Sleep(100 * time.Millisecond)
```

Comment:
Is the sleep necessary?

Reply:
It's aesthetic -- the mute was firing slightly before the voice had actually finished. It was bugging me. :)